client/app/components/EditVisualizationButton/QueryControlDropdown.jsx

```diff
@@ -75,7 +74,7 @@ export default function QueryControlDropdown(props) {
   );
 
   return (
-    <Dropdown trigger={["click"]} overlay={menu} overlayClassName="query-control-dropdown-overlay">
+    <Dropdown trigger={["click"]} overlay={menu} role="menu" overlayClassName="query-control-dropdown-overlay">
```

Comment:
I think the `Menu` is the only thing that needs the `menu` role. This doesn't need the `menu` role (I think it may [ignore the prop entirely](https://github.com/react-component/dropdown/blob/master/src/Dropdown.tsx#L7-L31), since `Dropdown`s aren't really on the DOM).